packages/react/src/Button/Button.module.css

```diff
@@ -11,31 +11,60 @@
   cursor: pointer;
   border: none;
   background: none;
+  display: inline-grid;
+  grid-template-areas: 'leadingIcon text trailingIcon';
 }
 
 .Button:focus:not(.Button--disabled[disabled]) {
   outline: none;
   box-shadow: 0 0 0 2px var(--brand-color-canvas-default), 0 0 0 5px var(--brand-color-focus);
 }
 
+.Button__text {
+  position: relative;
```

Comment:
What is `position: relative` used for? It doesn't seem to affect the Button's label layout.

Reply:
Yeah good spot, i'll remove this. I was trying to visually offset the text using `top` because the Mona Sans glyphs render off-centre right now. I ended up leaving it because I think it's been resolved in the latest typeface, which I'll upgrade to in a separate PR. 

E.g.
![Screenshot 2023-02-21 at 10 30 56](https://user-images.githubusercontent.com/13340707/220320322-0fc20558-4eb5-467b-a111-df71a2ed1c36.png)

Reply:
Update: I don't think the latest typeface will fix this. I've put back the updated `top` value offset, which helps improve alignment. LMK if you have a better way to do this @danielguillan.

![Screenshot 2023-02-21 at 10 39 13](https://user-images.githubusercontent.com/13340707/220322788-5d9ca906-57bd-4a91-b1cd-6b7102a967f0.png)